corehq/apps/users/views/mobile/users.py

```diff
@@ -1350,6 +1355,8 @@ def _excel_data(self, rows):
 @require_can_edit_commcare_users
 def count_users(request, domain):
     from corehq.apps.users.dbaccessors.all_commcare_users import get_commcare_users_by_filters
+    if not FILTERED_BULK_USER_DOWNLOAD.enabled_for_request(request):
+        return HttpResponseForbidden()
```

Comment:
I think this should raise a 404 to be consistent with `StaticToggle.required_decorator`.

Reply:
https://github.com/dimagi/commcare-hq/pull/27625/commits/91cee03bb0e8e9ce5964797c4d4465ea1d6e217f